### ISSUE TITLE
Consider adding a "skip to main content" link

### ISSUE THREAD

**Issue by TimJentzsch** [MurdoMaclachlan/website#38]
This is not a necessary addition, but might be a fun exercise and further improves the accessibility (and it's satisfying to see once it works).

The basic idea is that a keyboard user might not always want to tab through the whole navigation menu before reaching the main content (a screen-reader user doesn't have this problem as they have shortcuts to navigate more efficiently, for example by headers).
To fix this, a "Skip to main content" link can be added, that navigates the user directly to the `<main>` element which should contain the content of the page.

Just adding a plain link like this might confuse users who don't use keyboard navigation and might look ugly.
To solve this, the link can be hidden off-screen and then made visible only on keyboard focus.
(It's important to not hide it via `visibility: hidden` or `display: none` as it might not be focusable in this case)

So you can add the link to your navbar, give it a fixed height (or width) and set the default styling (e.g. margin) to move it off-screen to the top (or left).
Then you add `:focus` styling to move it back to the "normal" position.

You can read more about the technique [on WebAIM](https://webaim.org/techniques/skipnav/).

You can find examples on the WebAIM page itself, GitHub also has one (although an ugly one).
